src/http/ws_parser.h

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "enclave/tls_endpoint.h"
+#include "http_proc.h"
+
+#include <algorithm>
+#include <cctype>
+#include <endian.h>
+#include <string>
+
+namespace ws
+{
+  static constexpr size_t INITIAL_READ = 2;
+  static constexpr size_t OUT_CCF_HEADER_SIZE =
+    sizeof(uint16_t) + sizeof(size_t) * 3;
```

Comment:
Is it `* 3` here because of commit, term and global commit? Might be worth documenting this more clearly.

Reply:
Yes, will do.